pkg/sql/plan/build_test.go

```diff
@@ -505,6 +506,7 @@ func TestSingleTableSQLBuilder(t *testing.T) {
 
 		"SELECT DISTINCT N_NAME FROM NATION GROUP BY N_REGIONKEY", //test distinct with group by
 		"SELECT DISTINCT N_NAME FROM NATION ORDER BY N_REGIONKEY", //test distinct with order by
+		"select count(n_name) from nation limit 10 for update",
```

Comment:
What does this mean?     select update on an aggregate?    We lock everything in nation?

Reply:
no, wo not support select egg now, we will get a err info